### PR TITLE
Add docs

### PR DESCRIPTION
**Pull request by richlander**
- Add "official docs" page
- Add README.md as a docs index

### CODE REVIEW

File: Documentation/official-docs.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Official Microsoft Docs
+
+Microsoft maintains official documentation for .NET. You can see a list of those below:
+
+- [.NET Framework 4.6+](https://msdn.microsoft.com/library/w0x726c2(v=vs.110).aspx)
+- [.NET Core](https://docs.microsoft.com/dotnet/)
```
Comment (svick) [inline Documentation/official-docs.md]: https://docs.microsoft.com/dotnet/ links to documentation for both .Net Core and .Net Framework.

So, saying that it's just for .Net Core is inaccurate and I think that linking to .Net Framework documentation too is redundant.

Comment (richlander) [inline Documentation/official-docs.md]: Good point. Clarified the docs. Does that help?

Comment (svick) [inline Documentation/official-docs.md]: Yeah, the new version looks good to me.

File: Documentation/official-docs.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+Microsoft maintains official documentation for .NET, listed below. Both .NET Core and .NET Framework docs are relevant to .NET Core developers, both because many .NET Framework APIs are part of .NET Core (most relevant for .NET Core 2.0) but also because you can reference .NET Framework libraries with .NET Standard 2.0.
+
+## .NET Core
+
+- [Conceptual docs](https://docs.microsoft.com/dotnet/articles/core/)
+- [API docs](https://docs.microsoft.com/dotnet/core/api/)
+
+## .NET Framework
+
+- [.Conceptual docs](https://msdn.microsoft.com/library/w0x726c2.aspx)
```
Comment (svick) [inline Documentation/official-docs.md]: Nit: stray period before "Conceptual".

**Comment by Petermarcu** [comment]
@richlander, LGTM after @svick's nit is fixed.

**Comment by mairaw** [comment]
Fixed the small nit for @richlander 😄

**Pull request merged**